### ISSUE TITLE
Setup request - 400 invalid argument

### ISSUE THREAD

**Issue by DavidAnderson684** [jay0lee/got-your-back#303]
Using version 1.40 in a freshly unpacked directory:

```
$ ./got-your-back-1.40/gyb.py --action create-project --email example@gmail.com

Go to the following link in your browser:

        https://gyb-shortn.jaylee.us/wq4uh3

Enter verification code: 4/1AY.....(snip)
Creating project "Got Your Back Project"...
Checking project status...
 enabling API drive.googleapis.com...
 enabling API gmail.googleapis.com...
 enabling API groupsmigration.googleapis.com...
 enabling API iap.googleapis.com...
 enabling API vault.googleapis.com...
Creating Service Account
Setting project consent screen...

400: Request contains an invalid argument. - badRequest
```

**Comment by peddie** [comment]
I'm seeing this same problem.  I'm running
```
Got Your Back 1.40
https://git.io/gyb
Jay Lee - jay0lee@gmail.com
Python 3.9.1 64-bit final
google-api-client 1.12.8
Linux-5.10.0-1-amd64-x86_64-with-glibc2.31 x86_64
Path: /home/peddie/bin/gyb
OpenSSL 1.1.1i  8 Dec 2020
www.googleapis.com connects using TLSv1.3 TLS_AES_256_GCM_SHA384
```

**Comment by jay0lee** [comment]
This should be fixed now with 1.41 release:

git.io/gyb-releases
